scripts/issue-genesis-utxo-expenditure.sh

```diff
@@ -1,15 +1,27 @@
 #!/usr/bin/env bash
 
-RUNNER=${RUNNER:-cabal new-run -v0 --}
+if [ -e configuration/GenesisFiles/ ]
+then
+    :
+else
+    echo "configuration/GenesisFiles/GENHASH does not exist"
+	/bin/bash ./scripts/genesis.sh
```

Comment:
/bin/bash does not exist on every platform, that's why this script is launched via `/usr/bin/env bash`
isn't the `script/genesis.sh` executable?